pkg/pillar/cas/containerd.go

```diff
@@ -1141,3 +1156,16 @@ func getJSON(x interface{}) (string, error) {
 func GetRoofFsPath(rootPath string) string {
 	return filepath.Join(rootPath, containerRootfsPath)
 }
+
+// getEVEDownloadedLabel returns the label eve-downloaded = true which can be set on images or blobs
+// This label can be applied on image and blobs downloaded by eve. In kubevirt eve some images are downloaded by k3s.
+// We will differentiate those by the presence of this label.
+// This label will be used in Garbage collection of eve downloaded images
+// NOTE: This label will be set for all images and blobs newly created on kvm eve too.
```

Comment:
On kvm/xen etc, will you rely on this label for garbage collection? Or ignore it to avoid GCing blobs that were downloaded by older EVE releases?

Reply:
Right now in this PR, I don't use that label on kvm/xen flavors for GCing. Though we set the label on all newly created images/blobs. So nothing changed for kvm/xen as far as GC is concerned.